modal-go/examples/sandbox-volume/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log"
+
+	"github.com/modal-labs/libmodal/modal-go"
+)
+
+func main() {
+	ctx := context.Background()
+
+	app, err := modal.AppLookup(ctx, "libmodal-example", &modal.LookupOptions{
+		CreateIfMissing: true,
+	})
+	if err != nil {
+		log.Fatalf("Failed to lookup app: %v", err)
+	}
+
+	image, err := app.ImageFromRegistry("alpine:3.21", nil)
+	if err != nil {
+		log.Fatalf("Failed to create image: %v", err)
+	}
+
+	volume, err := modal.VolumeFromName(ctx, "libmodal-example-volume", &modal.VolumeFromNameOptions{
+		CreateIfMissing: true,
+	})
+	if err != nil {
+		log.Fatalf("Failed to create volume: %v", err)
+	}
+
+	writerSandbox, err := app.CreateSandbox(image, &modal.SandboxOptions{
+		Command: []string{
+			"sh",
+			"-c",
+			"echo 'Hello from writer sandbox!' > /mnt/volume/message.txt",
+		},
+		Volumes: map[string]*modal.Volume{
```

Comment:
Question to Go people - is this an idiomatic/clean way to specify the mounts or would some other structure be preferable? This is at least in line with the Python/JS way we do it...

Reply:
lgtm